src/Foundation/Enum.cs

```diff
@@ -1115,6 +1118,7 @@ public enum NSURLRelationship : nint {
 	}
 
 	// NSTextCheckingResult.h:typedef NS_OPTIONS(uint64_t, NSTextCheckingType)
```

Comment:
just a note but `NS_OPTIONS` means we could, in some cases, have xtro help to spot *some* enums (there's so much ways they are defined that's it won't be foolproof, otoh it should not create false positives).